frontend/simple/js/event-log.js

```diff
@@ -0,0 +1,91 @@
+import localforage from 'localforage'
```

Comment:
Why is this file called `event-log.js`? It was renamed to `database.js` in `master`. Seems like some changes weren't remerged?